### PR TITLE
feat: add nested section, switch field and label on check boxes

### PR DESCRIPTION
**Pull request by Mahaveer1141**
## What does this change do?
- add nested section, switch field, and label on checkboxes

## Why is this change needed?
- nested section and switch field were required.
- custom label on checkbox groups.

## How were the changes done?
- created a new field on section `nested_section` for the nested section, which works similarly to the section.
- add new field type `switch` on form fields.

https://github.com/user-attachments/assets/bf54470a-6cb2-44e3-9891-8361304c7781

### CODE REVIEW

File: .rubocop-https---raw-githubusercontent-com-commutatus-cm-linters-main-rubocop-yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 AllCops:
-   DisabledByDefault: false
+  DisabledByDefault: false
+  Exclude:
+    - 'db/migrate/*'
```
Comment (anbublacky) [inline .rubocop-https---raw-githubusercontent-com-commutatus-cm-linters-main-rubocop-yml:4]: Remove this file bro. This needs to be written on the common linter

File: app/assets/javascripts/cm_admin/form_validation.js [hunk 2]
```diff
@@ -1,35 +1,53 @@
...
   });
   if (submit.length === 0) {
-    $('.' + form_class).submit();
-    return $('[data-behaviour="form_submit"]').button('loading');
+    $("." + form_class).submit();
+    return $('[data-behaviour="form_submit"]').button("loading");
+  }
+});
+
+$(document).on("change", $(".multiple-checkbox"), function (e) {
```
Comment (anbublacky) [inline app/assets/javascripts/cm_admin/form_validation.js:45]:
```suggestion
$(document).on("change", ".multiple-checkbox", function (e) {
```

File: app/assets/stylesheets/cm_admin/base/form.scss [hunk 3]
```diff
@@ -21,9 +21,16 @@
       @extend .h6, .fw-semibold;
       color: $primary-text-clr;
     }
+    .nested-form-title {
+      color: #21252980;
```
Comment (anbublacky) [inline app/assets/stylesheets/cm_admin/base/form.scss:25]: Seems wrong here and use a variable

**Review by anbublacky — CHANGES_REQUESTED** [review]

File: .rubocop-https---raw-githubusercontent-com-commutatus-cm-linters-main-rubocop-yml (hunk 1 above)
Comment (Mahaveer1141) [reply .rubocop-https---raw-githubusercontent-com-commutatus-cm-linters-main-rubocop-yml:4]: @anbublacky are you referring to remove the changes from this file, or remove the whole file? this file was already committed before.

File: app/assets/javascripts/cm_admin/form_validation.js (hunk 2 above)
Comment (Mahaveer1141) [reply app/assets/javascripts/cm_admin/form_validation.js]: done

File: app/assets/javascripts/cm_admin/form_validation.js [hunk 4]
```diff
@@ -1,35 +1,53 @@
...
   });
   if (submit.length === 0) {
-    $('.' + form_class).submit();
-    return $('[data-behaviour="form_submit"]').button('loading');
+    $("." + form_class).submit();
+    return $('[data-behaviour="form_submit"]').button("loading");
+  }
+});
+
+$(document).on("change", ".multiple-checkbox", function (e) {
```
Comment (mikevic) [inline app/assets/javascripts/cm_admin/form_validation.js:45]: @Mahaveer1141 - let's use data-behaviours here too?

**Review by mikevic — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/cm_admin/form_validation.js (hunk 4 above)
Comment (Mahaveer1141) [reply app/assets/javascripts/cm_admin/form_validation.js]: done

**Pull request merged**